src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,35 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
-
+        try (BufferedReader reader = new BufferedReader(new FileReader(fromFileName))) {
+            int buy = 0;
+            int supply = 0;
+            String line;
+            String[] data;
+            while ((line = reader.readLine()) != null) {
+                data = line.split(",");
+                switch (data[0]) {
+                    case "buy":
+                        buy += Integer.parseInt(data[1]);
+                        break;
+                    case "supply":
+                        supply += Integer.parseInt(data[1]);
+                }
```

Comment:
There is no default case in the switch statement to handle unexpected data types. Consider adding a default case to handle any unexpected values in the data array.